src/apps/Permissions/EntityPermissions.js

```diff
@@ -26,12 +30,14 @@ class EntityPermissions extends React.PureComponent {
       return null
     }
 
+    console.log('ALLELE?', resolveEntity(entityAddress, daoAddress))
```

Comment:
We can probably remove this, although I am curious what this translates to :)

Reply:
True randomness generated by my fingers 😆